app/uk/gov/hmrc/lisaapi/controllers/AccountController.scala

```diff
@@ -98,6 +112,24 @@ class AccountController extends LisaController with LisaConstants {
     }
   }
 
+  private def processGetAccountDetails(lisaManager:String, accountId:String)(implicit hc: HeaderCarrier,startTime:Long) = {
+    service.getAccount(lisaManager, accountId).map { result =>
+      result match {
+        case response : GetLisaAccountSuccessResponse  => {
+          LisaMetrics.incrementMetrics(startTime,LisaMetricKeys.ACCOUNT)
+          Ok(Json.toJson(response))
+        }
+
+
+        case _ => {
```

Comment:
We need to specifically match for account not found and return it. Our default fallback would be to return an INTERNAL_SERVER_ERROR.

I've written an test case for this in the acceptance tests and created a sub task to update the code.